### DISCUSSION TITLE
How to catch 413 HTTP error using Sentry?

### DISCUSSION THREAD

**Discussion by Vadorequest** [vercel/community#727] [Help] [answered]
I'm calling one of my Next.js API endpoint and sending a file that's over 5MB (apparently, because I'm getting a 413 error code).

I'm using Sentry, but it doesn't catch this particular error. Maybe because it's happening even before the function's code is called?

How do I catch those?

**Comment by swarnava** [comment, marked as answer]
This sounds like an issue with [Sentry SDK](https://forum.sentry.io/t/413-request-entity-too-large/6359). Have you tried reaching out to their support team?

**Reply by Vadorequest** [reply]
I think it's rather related to an actual Vercel bug.

See https://github.com/vercel/vercel/discussions/8175

The function's code is indeed not called at all, which explains why Sentry didn't catch anything, the error happens before Sentry is even executed.

**Reply by swarnava** [reply]
Hi,

We are internally using Sentry and I can confirm that 413 errors are being reported correctly: 

<img width="468" alt="image" src="https://user-images.githubusercontent.com/1162991/179492782-db83d1f0-b70f-4eac-b2f8-d8972237e2f8.png">

**Reply by Vadorequest** [reply]
Thanks for proving me wrong! I'll have to tackle this again then, the old template I used might not be up-to-date with Sentry/Next best practices.

**Reply by swarnava** [reply]
No problem 🙏
